src/components/thumbnail-card/__tests__/__snapshots__/ThumbnailCardDetails.test.js.snap

```diff
@@ -1,104 +1,451 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`components/thumbnail-card/ThumbnailCardDetails should render 1`] = `
-<div
-  className="thumbnail-card-details"
->
-  <div
-    className="thumbnail-card-details-content"
-  >
+{
```

Comment:
if we are switching to RTL, we need to remove these snapshots and update the tests that are doing the `toMatchSnapshot` to look for aspects of the thumbnail via RTL instead